packages/react-reconciler/src/ReactFiberTreeReflection.js

```diff
@@ -10,6 +10,7 @@
 import type {Fiber} from './ReactInternalTypes';
 import type {Container, SuspenseInstance} from './ReactFiberHostConfig';
 import type {SuspenseState} from './ReactFiberSuspenseComponent.old';
+import {isRendering as ReactCurrentFiberIsRendering} from './ReactCurrentFiber';
```

Comment:
This is actually a bit of a red flag because for third-party renderers, `/reflection` entry point won't "see" the value. So maybe this is a good motivation to move `isRendering` into `ReactDebugCurrentFrame`.